templates/managed/apps/index.html

```diff
@@ -16,7 +16,7 @@
       <h1>Performant open source with Managed Apps</h1>
       <p class="p-heading--4">One vendor, easy deployment, guaranteed uptime</p>
       <p>
-        <a href="/managed/contact-us" class="p-button--positive js-invoke-modal" data-testid="interactive-form-link" data-form-location="/shared/forms/interactive/managed-apps.html" data-form-id="3545" data-lp-id="3828" data-return-url="https://ubuntu.com/managed/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Free deployment assessment</a>
+        <a href="/managed/contact-us" class="p-button--positive js-invoke-modal" data-testid="interactive-form-link" data-form-location="/shared/forms/interactive/managed-apps.html" data-form-id="3545" data-lp-id="3828" data-return-url="https://ubuntu.com/managed/thank-you#success" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Free deployment assessment</a>
```

Comment:
I think it needs to return to the original page with `#success` rather than going to a thank-you page 

```suggestion
        <a href="/managed/contact-us" class="p-button--positive js-invoke-modal" data-testid="interactive-form-link" data-form-location="/shared/forms/interactive/managed-apps.html" data-form-id="3545" data-lp-id="3828" data-return-url="https://ubuntu.com/managed/apps#success" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Free deployment assessment</a>
```